src/coreclr/tests/src/readytorun/crossboundarylayout/crossboundarytests.cmd

```diff
@@ -0,0 +1,322 @@
+echo off
```

Comment:
I wonder if this script (and the other ones) should be located in src/coreclr/tests/scripts instead.